solidity/dashboard/src/services/token-grants.service.js

```diff
@@ -53,6 +59,26 @@ const getGrantDetails = async (
   isManagedGrant = false
 ) => {
   const { yourAddress } = web3Context
+  const escrowDepositsEvents = await fetchEscrowDepositsByGrantId(grantId)
+  const escrowOperatorsToWithdraw = []
+  let escrowWithdrawableAmount = 0
+  let escrowWithdrawTotalAmount = 0
+
+  for (const event of escrowDepositsEvents) {
+    const {
+      returnValues: { operator },
+    } = event
+    const withdrawable = await fetchWithdrawableAmountForDeposit(operator)
+    const withdraw = await fetchDepositWithdrawnAmount(operator)
+
+    escrowWithdrawTotalAmount = add(escrowWithdrawTotalAmount, withdraw)
+
+    if (gt(withdrawable, 0)) {
```

Comment:
wouldn't it be enough to `if(withdrawable)`? I think `falsy` result handles 0

Reply:
`withdrawable` is `string` so `if(withdrawable)` where `withdrawable -> "0"` it's a non-empty string and evaluated to true.
I think we can just `if(withdrawable !== "0")`, what do you think?

Reply:
oh, I didn't realize that `withdrawable` is a string, `0` in `gt(withdrawable, 0)` confused me. Anyway, I think it's fine then, let's leave it as is.